fraqbot/chatbot.py

```diff
@@ -15,6 +15,7 @@
 import yaml
 
 # from Local.aoc import AOC
+from Local.bible import Bible
```

Comment:
You need to add `Bible` as baseplate child as well (see bottom of `chatbot.py` for examples.)